qttypes/src/lib.rs

```diff
@@ -640,23 +641,105 @@ impl std::fmt::Debug for QString {
     }
 }
 
+// XXX: The Default, Clone and Copy traits are provided by cpp_class! but they won't have these
+// nice "Wrapper around ..." docs with links on them. How can we fix this?
```

Comment:
I don't think we need such documentation. It is obvious for any Rust developer what Default and Clone do. Also the documentation for trait isn't even well displayed in docs.rs

Reply:
While is true that redefinition of trait docs do not show up unless you explicitly expand them (+), nevertheless they are still there, safe and sound (no pun intended).
![20210423_232901](https://user-images.githubusercontent.com/6737986/115926633-d5d75300-a48b-11eb-83cd-c21f75c5a232.png)

> I think that's fine. These links wouldn't really help anyone IMHO.

Links may not be _that_ important, but (as I mentioned in some other thread) literal copy-pasted signature is essential for someone who looks for a Rust replacement of a particular piece of Qt API. It would be much less of a problem, it things were possible to map 1:1, but instead in Rust we have From traits, absence of constructors and default arguments, different names for basic types (double vs f64) etc. — all of which causes difficulties in navigation.

Reply:
Without assuming much knowledge of Qt from the users of this crate, I'd be happy to mention in Clone docs about implicit sharing of the majority of datatypes directly supported by QVariant. But now that I think about it, such info might fit in QVariant's main doc comment itself.